src/app/seamlyme/dialogs/dialogmdatabase.ui

```diff
@@ -24,6 +24,32 @@
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
+   <item>
```

Comment:
This overwrites class QVBoxLayout. Would it be better to create a new class?

Reply:
Sorry, this is my first time with C++ or creating UI with Qt. I don't know enough to know what the change is you're asking for. I looked around the code but couldn't make sense of if/how/why we should change that layout class from a QVBoxLayout? 

Are you saying like, make a new class in code that subclasses from QVBoxLayout and has some built-in functionality? If yes, what class-specific functionality are you thinking of?

If you think it would be an improvement, I'll do it, I'll just need direction

Reply:
>  This overwrites class QVBoxLayout. Would it be better to create a new class?

This makes no sense.  [D]ialogmdatabase.ui is a Creator generated ui file (or should be as you should not be editing ui files directly), and there are only  4 - "layout" options to choose from.  Unless you want to subclass a new widget and either promote it or make it a plugin....  I don't understand the point. 

That said, I can see that a  QHBoxLayout was added inside a QFrame inside the QVBoxLayout... which is perfectly acceptable, as you (generally) can nest frames, groupboxes, scroll areas, layouts, etc. Is the added QHBoxLayout  necessary? Without looking at the form I can't say.

Reply:
> > This overwrites class QVBoxLayout. Would it be better to create a new class?
> 
> This makes no sense. [D]ialogmdatabase.ui is a Creator generated ui file

Agreed, this is a generated file. Just forgot that I was dealing with Qt.

Reply:
@not-Ian:
> this is my first time with C++ or creating UI with Qt.
You're doing great. No need to change anything in this file.